jung-visualization/src/main/java/edu/uci/ics/jung/visualization/BasicVisualizationServer.java

```diff
@@ -304,7 +315,48 @@ protected void renderGraph(Graphics2D g2d) {
     newXform.concatenate(
         renderContext.getMultiLayerTransformer().getTransformer(Layer.VIEW).getTransform());
 
+    AffineTransform spatialTransform = new AffineTransform(oldXform);
+    spatialTransform.concatenate(
+        renderContext.getMultiLayerTransformer().getTransformer(Layer.VIEW).getTransform());
+    spatialTransform.concatenate(
+        renderContext.getMultiLayerTransformer().getTransformer(Layer.LAYOUT).getTransform());
+
+    try {
+      spatialTransform = spatialTransform.createInverse();
+    } catch (Exception e) {
+      e.printStackTrace();
+    }
```

Comment:
I wonder if it would be better to let exceptions thrown here to propogate to the caller.